thunder/core/prims.py

```diff
@@ -2216,6 +2217,20 @@ def _make_elementwise_unary_prim(
     output_dtype_kind=ELEMENTWISE_PRIM_OUTPUT_DTYPE_KIND.INT_FOR_NUMBER,
 )
 
+
+def frexp_meta(a: TensorProxy, *, out: None | TensorProxy = None) -> (TensorProxy, TensorProxy):
```

Comment:
While it looks a like a few primitives do have an `out` keyword, they probably shouldn't. We should take `out` away from the primitive

Reply:
Got it! Thank you for sharing the details!

I'll remove the `out` arg from here and submit another PR to remove it from the operators that support it.